roles/splunk/tasks/configure_dmc.yml

```diff
@@ -0,0 +1,21 @@
+---
+# Add all hosts to the DMC except indexer cluster hosts as per the docs
+- name: Configure systems as search peers to be monitored except indexers
+  ansible.builtin.shell: |
+    {{ splunk_home }}/bin/splunk add search-server https://{{ item }}:{{ splunkd_port }} -auth "{{ splunk_auth }}" -remoteUsername "{{ splunk_admin_username }}" -remotePassword "{{ splunk_admin_password }}"
+  loop: "{{ query('inventory_hostnames', 'all:!indexer') }}"
```

Comment:
Nit pick: If there are multiple IDXC in the same inventory (I sometimes have), this will loop through all indexers in all clusters, and not only the one belonging to that IDXC. But this may be an edge case.

Reply:
It would loop through all Cluster masters, but not all indexers, hence `all:!indexer`, right?